package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Read a protobuf schema from the disk, parse it and resolve all imports",
   "main": "index.js",
   "dependencies": {
+    "lodash": "^4.8.2",
```

Comment:
I think we should avoid adding a new dependency here. What about hand-writing the necessary functions?